src/utils/common.ts

```diff
@@ -83,6 +83,67 @@ export function getOverrideIP() {
     .trim();
 }
 
+/**
+ * Here we check that the value in the custom param can be considered to be True:
+ * 1. Boolean true value
+ * 2. Any number !== 0
+ * 3. 'yes' or 'true' string in lower / upper case
+ */
+export const hasTrueValue = (value: undefined | null | string | number | boolean): boolean => {
+  // null, undefined or empty string
+  if (!value) {
+    return false;
+  }
+
+  if (typeof value === 'boolean') {
+    return value;
+  }
+
+  // 0 equals to false, other numbers equal to true
+  if (typeof value === 'number') {
+    return Boolean(value);
+  }
+
+  if (!isNaN(Number(value))) {
+    return Number(value) > 0;
+  }
+
+  const strValue = value.toLowerCase();
+
+  return strValue === 'true' || strValue === 'yes';
+};
+
+/**
+ * @deprecated We should use true and hasTrueValue check for new properties
```

Comment:
I wouldn't mark the code for this as deprecated. The _usage_ of this for hasSubscription is deprecated, but the code itself is fine.

Reply:
My suggestion was to use only one approach: either `isTruthyCustomParamValue` or `isFalsyCustomParamValue`.
I think it may be confusing if we use an opposite (false) value for some cases. It seems to me that 'True' could solve all our cases, so instead of `requiresSubscription` we use `free` and instead of `useRealService` we use `useSandbox`.

Reply:
Based off of the check in today, it sounds like we can get rid of requires subscription so also can get rid of this method. Though I will say, there is something to be said for having both. Because we are being careful about what means truthy or falsey, they aren't exact opposites.  Example 'Danny' is neither truthy nor falsey. So if we have a case like requires subscription where it's a false value, it won't be a good solution to just put !isTruthy.